e2e/__tests__/circusDeclarationErrors.test.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {skipSuiteOnJasmine} from '@jest/test-utils';
+import runJest from '../runJest';
+
+skipSuiteOnJasmine();
+
+it('defining tests and hooks asynchronously throws', () => {
+  const result = runJest('circus-declaration-errors', [
+    'asyncDefinition.test.js',
+  ]);
+
+  expect(result.status).toBe(1);
+  expect(result.stderr).toMatchSnapshot();
```

Comment:
you need to filter out timings.

```ts
import {wrap} from 'jest-snapshot-serializer-raw';
import {cleanup, extractSummary, writeFiles} from '../Utils';

const result = runJest('circus-declaration-errors', [
    'asyncDefinition.test.js',
]);

const {rest} = extractSummary(result.stderr);

expect(wrap(rest)).toMatchSnapshot();
```

Reply:
you can snapshot `summary` as well if you want - it'll have timestamp replaced with a static value

Reply:
Thanks, changed